examples/sentiment_analysis_example.py

```diff
@@ -0,0 +1,90 @@
+"""Copyright 2019 REV
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Why is this the Apache license vs the MIT license?

Reply:
Our examples are all under apache. I don't know the distiction. I don't think it really matters